src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -99,4 +136,152 @@ private void copyUrl() {
+        if (keyEvent.getCode() == KeyCode.ESCAPE) {
+            this.close();
+        }
+    }
+
+    /**
+     * Changes the focus to the card before the current
+     */
+    public void focusPreviousCard(CommandHelpCard card) {
+        int idx = commandHelpCardsContainer.getChildren().indexOf(card.getRoot());
+        if (idx > 0) {
+            commandHelpCardsContainer.getChildren().get(idx - 1).requestFocus();
+        }
+    }
+
+    /**
+     * Changes the focus to the card after the current
+     */
+    public void focusNextCard(CommandHelpCard card) {
+        int idx = commandHelpCardsContainer.getChildren().indexOf(card.getRoot());
+        int size = commandHelpCardsContainer.getChildren().size();
+        if (idx != -1 && idx < size - 1) {
+            commandHelpCardsContainer.getChildren().get(idx + 1).requestFocus();
+        }
+    }
+
+    private void setHelpCards(ArrayList<CommandHelpCard> helpCards) {
+        commandHelpCardsContainer.getChildren().removeAll();
+        for (CommandHelpCard helpCard : helpCards) {
+            commandHelpCardsContainer.getChildren().add(helpCard.getRoot());
+        }
+    }
+
+    /**
+     * Returns help cards for default window
+     */
+    private ArrayList<CommandHelpCard> defaultHelpCards() {
+        ArrayList<CommandHelpCard> helpCards = new ArrayList<>();
+
+        CommandHelpCard addCommandCard = new CommandHelpCard(
+            AddCommand.COMMAND_WORD,
+            AddCommand.MESSAGE_USAGE_PERSON,
+            AddCommand.ADD_PERSON_EXAMPLE
+        );
+        helpCards.add(addCommandCard);
+
+        CommandHelpCard clearCommandCard = new CommandHelpCard(
+            ClearCommand.COMMAND_WORD, "Clears address book", "clear"
+        );
+        helpCards.add(clearCommandCard);
+
+        CommandHelpCard deleteCommandCard = new CommandHelpCard(
+            DeleteCommand.COMMAND_WORD, DeleteCommand.MESSAGE_USAGE, "delete 1"
+        );
+        helpCards.add(deleteCommandCard);
+
+        CommandHelpCard editCommandCard = new CommandHelpCard(
+            EditCommand.COMMAND_WORD, EditCommand.MESSAGE_USAGE, EditCommand.PERSON_EXAMPLE
+        );
+        helpCards.add(editCommandCard);
+
+        CommandHelpCard exitCommandCard = new CommandHelpCard(
+            ExitCommand.COMMAND_WORD, "Closes the app", "exit"
+        );
+        helpCards.add(exitCommandCard);
+
+        CommandHelpCard findCommandCard = new CommandHelpCard(
+            FindCommand.COMMAND_WORD, FindCommand.MESSAGE_USAGE, "find bob"
+        );
+        helpCards.add(findCommandCard);
+
+        CommandHelpCard helpCommandCard = new CommandHelpCard(
+            HelpCommand.COMMAND_WORD, HelpCommand.MESSAGE_USAGE, "help"
+        );
+        helpCards.add(helpCommandCard);
+
+        CommandHelpCard inspectCommandCard = new CommandHelpCard(
+            InspectCommand.COMMAND_WORD, InspectCommand.MESSAGE_USAGE, "inspect"
+        );
+        helpCards.add(inspectCommandCard);
```

Comment:
Should the example "inspect" contain the index of the contact to inspect after?
For example, "inspect 1".

Reply:
Yep. Thanks for pointing it out. Fixed it.